### ISSUE TITLE
超参数

### ISSUE THREAD

**Issue by wuyueshijiu** [Qingqing02/HGCLD#1]
作者您好，方便您分享下最优超参数的配置吗，这样方便我复刻您的工作，以下是我复现时的超参数：
- Dataset: Tmall
- General Training:
- --lr (Learning Rate): 1e-3 (0.001)
- --batch (Batch Size): 512
- --reg (L2 Regularization): 4e-2 (0.04)
- --latdim (Embedding Size): 128
- --gcn_layer (GCN Layers): 3
- Diffusion Model:
- --difflr (Diffusion LR): 1e-3 (0.001)
- --dims (Denoise Network): [64]
- --steps (Diffusion Steps): 100
- --d_emb_size (Time Embedding): 8
- --noise_scale : 1e-4 (0.0001)
- Contrastive Learning:
- --contrast_weight : 0.1
- --temperature : 0.1
Test: Recall = 0.0511, NDCG = 0.0221
结果和文章中相差较大
